packages/storybook-docs/src/globalStyles.scss

```diff
@@ -1,410 +1,409 @@
 @use '~@talend/design-tokens/lib/tokens';
 
 * {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
 }
 
 html {
-  font-size: 62.5%;
-  scroll-behavior: smooth;
+	font-size: 62.5%;
+	scroll-behavior: smooth;
 
-  body {
-    font-size: 1rem;
-  }
+	body {
+		font-size: 1rem;
+	}
 }
 
 #docs-root {
-  margin-top: 0;
-  font: tokens.$coral-paragraph-m;
-  color: tokens.$coral-color-neutral-text;
+	margin-top: 0;
+	font: tokens.$coral-paragraph-m;
+	color: tokens.$coral-color-neutral-text;
+	z-index: tokens.$coral-elevation-layer-flat;
+	position: relative;
```

Comment:
Real change is here 
The root of the storybook is at the first layer of z-index and relative